client/me/purchases/payment-methods/style.scss

```diff
@@ -139,15 +139,23 @@
 
 .payment-method-backup-toggle {
 	grid-area: payment-method-backup;
-	margin-left: 76px;
-	margin-top: 6px;
+	margin-inline-start: 76px;
+	margin-top: 16px;
 	width: 100%;
 
+	& .components-checkbox-control [data-wp-component="HStack"] {
```

Comment:
we shouldn't target the internals of components, it's going to be very brittle

Reply:
@jjchrisdiehl Matías is right. Can you open a follow-up PR to fix this?

Reply:
I can reinstate https://github.com/Automattic/wp-calypso/pull/93757

Reply:
Agreed. #93757 appears to be a better fix.

Reply:
cc @Automattic/team-calypso: can we make it clearer in `@wordpress/components` documentation that the classnames are internals and not part of the public API, and as such they are subject to change and shouldn't be used?

Reply:
A "best practices" guide for consumers is in the works, in which we will cover CSS.